caffe2/python/examples/resnet50_trainer.py

```diff
@@ -258,22 +261,40 @@ def Train(args):
 
     # Model building functions
     def create_resnet50_model_ops(model, loss_scale):
-        [softmax, loss] = resnet.create_resnet50(
-            model,
-            "data",
-            num_input_channels=args.num_channels,
-            num_labels=args.num_labels,
-            label="label",
-            no_bias=True,
-        )
+        initializer = (pFP16Initializer if args.fp16 else Initializer)
+        conv_tensor_core = (True if args.fp16 else False)
+        fc_engine = ('TENSORCORE' if args.fp16 else '')
```

Comment:
We could add a little sugar to the fc helper to clean this up. Happy to bundle that change with this PR if desired.